pyiceberg/table/__init__.py

```diff
@@ -382,7 +383,54 @@ def commit_transaction(self) -> Table:
             return self._table
 
 
+class CreateTableTransaction(Transaction):
+    @staticmethod
+    def create_changes(table_metadata: TableMetadata) -> Tuple[TableUpdate, ...]:
```

Comment:
I think we can also accumulate the changes in `_updates` of the `Transaction` itself